src/app/project-dialog/project-dialog.component.html

```diff
@@ -0,0 +1,56 @@
+<div class="col-md-12 col-sm-12 col-xs-12 createProjectBlock">
+    <div class="col-md-12 col-sm-12 col-xs-12">
+        <label for="projectname">Project Name</label>
```

Comment:
for this specific block I have two labels. the reason to take this out is the patternfly css overwriting issues.